### ISSUE TITLE
Merge localized resources

### ISSUE THREAD

**Issue by alexdyb** [dahall/TaskScheduler#10]
Hi,
I tried to merge TaskScheduler together with german localized resources, like here: https://taskscheduler.codeplex.com/discussions/351311
Added these 3:
de\Microsoft.Win32.TaskScheduler.resources.dll 
de\Microsoft.Win32.TaskSchedulerEditor.resources.dll 
de\TimeSpan2.resources.dll

ILSpy shows .de. resources are there in merged library:
![image](https://user-images.githubusercontent.com/32156905/31234271-28734e72-a9f7-11e7-881e-fa05084347cd.png)

But dialog is not localized. Perhaps i missed something?
I start my project with         
Dim ci As CultureInfo = New CultureInfo("de-DE")
Thread.CurrentThread.CurrentCulture = ci
Thread.CurrentThread.CurrentUICulture = ci

if I reference original libraries - localization is ok
Thanks

**Comment by dahall** [comment]
Which dialog is not localized? I have just run with the same CultureInfo settings and get all the standard dialogs in German. Are you putting those settings in your Main method before calling Application.Run?

**Comment by alexdyb** [comment]
The whole TaskEdit dialog is still on English. Yes, culture settings are the first after Sub Main(). If I set reference to original DLLs on same project - I get German. Just tried on latest release. 

Here my ILMerge file:

"C:\Program Files (x86)\Microsoft\ILMerge\ILMerge.exe" /t:library /targetplatform:v4 /keyfile:TaskService.snk /xmldocs /log:ts.log /allowDup /ndebug /out:ts.dll Microsoft.Win32.TaskScheduler.dll Microsoft.Win32.TaskSchedulerEditor.dll TimeSpan2.dll GroupControls.dll ActiveDirectoryObjectPicker.dll AeroWizard.dll de\Microsoft.Win32.TaskScheduler.resources.dll de\Microsoft.Win32.TaskSchedulerEditor.resources.dll de\TimeSpan2.resources.dll

I have replaced /union with /allowDup, else i get:
ILMerge: Writing target assembly 'ts.dll'.
An exception occurred during merging:
Unable to cast object of type 'System.Compiler.Field' to type 'System.Compiler.Method'.

I merging net4 files.
